django/db/backends/postgresql/operations.py

```diff
@@ -257,3 +258,21 @@ def window_frame_range_start_end(self, start=None, end=None):
                 'and FOLLOWING.'
             )
         return start_, end_
+
+    def explain_query_prefix(self, format=None, **options):
+        prefix = super().explain_query_prefix(format, **options)
+
+        extra = {}
+        if format:
+            extra['FORMAT'] = format
+
+        if options:
+            extra.update({
+                name.upper(): 'true' if value else 'false'
+                for name, value in options.items()
```

Comment:
I queried whether we needed to validate `options` here, but the DB does it for us. 

e.g. Using `SUMMARY` on Postgres <10: 

```
EXPLAIN (ANALYSE true, TIMING true, SUMMARY true) ...
ERROR:  unrecognized EXPLAIN option "summary"
```

There error message is clear enough.

Reply:
Yeah, I'd like to keep this lax if possible, it's better for the DB to validate it rather than us IMO. I'm going to try the `buildbot, test on oracle.` command and see if that works. Thanks for taking a look! 👍